<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Curve_Monitoring</class>
 <widget class="QWidget" name="Curve_Monitoring">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="Header" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>60</height>
    </rect>
   </property>
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>15</y>
      <width>155</width>
      <height>33</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="current_time">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>0</y>
      <width>310</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>yyyy-MM-dd hh:mm:ss</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="title">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>0</y>
      <width>180</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>曲线监视</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Main" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>1280</width>
     <height>660</height>
    </rect>
   </property>
   <widget class="QWidget" name="Temperature_PV_box" native="true">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>25</y>
      <width>237</width>
      <height>82</height>
     </rect>
    </property>
    <widget class="QLabel" name="temperature_pv_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>237</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  温度 PV</string>
     </property>
    </widget>
    <widget class="QLabel" name="temperature_pv_company_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QCheckBox" name="temperature_pv_checkbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="temperature_pv_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Temperature_SV_box" native="true">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>25</y>
      <width>237</width>
      <height>82</height>
     </rect>
    </property>
    <widget class="QLabel" name="temperature_sv_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>237</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  温度 SV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="temperature_sv_checkbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="temperature_sv_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="temperature_sv_company_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>°C</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Humidity_PV_box" native="true">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>25</y>
      <width>237</width>
      <height>82</height>
     </rect>
    </property>
    <widget class="QLabel" name="humidity_pv_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>237</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  湿度 PV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="humidity_pv_checkbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="humidity_pv_company_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="humidity_pv_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Humidity_SV_box" native="true">
    <property name="geometry">
     <rect>
      <x>1005</x>
      <y>25</y>
      <width>237</width>
      <height>82</height>
     </rect>
    </property>
    <widget class="QLabel" name="humidity_sv_label">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>237</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>  湿度 SV</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="humidity_sv_checkbox">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>25</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="humidity_sv_lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="humidity_sv_company_label">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>40</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>%</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_0">
    <property name="geometry">
     <rect>
      <x>48</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_1">
    <property name="geometry">
     <rect>
      <x>169</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>532</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_6">
    <property name="geometry">
     <rect>
      <x>775</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_7">
    <property name="geometry">
     <rect>
      <x>896</x>
      <y>528</y>
      <width>100</width>
      <height>55</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="storage_frequency_label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>存储频率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="storage_frequency_line">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>640</y>
      <width>110</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="storage_frequency_lineEdit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>605</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_spacing_label">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>存储空间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="storage_spacing_lineEdit">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>605</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="storage_spacing_line">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>640</y>
      <width>110</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cursor_time_label">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>600</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>光标时间：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cursor_time_lineEdit">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>605</y>
      <width>113</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="cursor_time_line">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>640</y>
      <width>110</width>
      <height>2</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="storage_frequency_pushButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>600</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QCustomPlot" name="CurveWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>1221</width>
      <height>411</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="Footer" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>720</y>
     <width>1280</width>
     <height>80</height>
    </rect>
   </property>
   <widget class="QPushButton" name="font_page_pbtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>目  录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="previous_page_pbth">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>上一页</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saving_pbtn">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>保存至U盘</string>
    </property>
   </widget>
   <widget class="QPushButton" name="real_time_pbtn">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>10</y>
      <width>145</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>实  时</string>
    </property>
   </widget>
   <widget class="QLabel" name="footer_line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>3</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="status_box" native="true">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>10</y>
      <width>431</width>
      <height>61</height>
     </rect>
    </property>
    <widget class="QLabel" name="status_box_line">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>1</y>
       <width>2</width>
       <height>59</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_text">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>定值运行</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="status_1">
     <property name="geometry">
      <rect>
       <x>175</x>
       <y>15</y>
       <width>25</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_2">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>15</y>
       <width>22</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_3">
     <property name="geometry">
      <rect>
       <x>255</x>
       <y>15</y>
       <width>15</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_4">
     <property name="geometry">
      <rect>
       <x>285</x>
       <y>18</y>
       <width>30</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>16</y>
       <width>27</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="status_6">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>15</y>
       <width>30</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
